pkgs/development/tools/build-managers/bazel/default.nix

```diff
@@ -71,7 +92,7 @@ stdenv.mkDerivation rec {
 
   # Build the CPP and Java examples to verify that Bazel works.
 
-  doCheck = true;
+  doCheck = false;
```

Comment:
This should be a conditional, I think.

Reply:
I'll change this to `(!stdenv.isDarwin)` if I can't get tests working